homeassistant/components/aws/manifest.json

```diff
@@ -2,6 +2,10 @@
   "domain": "aws",
   "name": "Amazon Web Services (AWS)",
   "documentation": "https://www.home-assistant.io/integrations/aws",
-  "requirements": ["aiobotocore==0.11.1"],
-  "codeowners": ["@awarecan"]
-}
+  "requirements": [
+    "aiobotocore==0.11.1"
+  ],
+  "codeowners": [
+    "@awarecan"
```

Comment:
You need to add your self also to code owner

Reply:
Added myself as codeowner.